RELEASE_NOTES/3.50.md

```diff
@@ -0,0 +1,20 @@
+App
+
+- Add a `g` hotkey to dev to open GraphiQL in your browser, using the same credentials as your app.
+- Javy must only be installed in case the app includes at least one js function
+- Running config commands works properly when they are run outside the app root folder
+- Uuid is no longer generated during app deploy because it's generated in the backend, instead.
+- Add support to test locales with functions in Dev mode.
+- Preserve proxy url subpath when running dev
+- Allow enabling and disabling dev store preview mode with dashboard managed extensions.
+- Javascript functions/extensions now support environment variable sustitution.
+- Add support for different flavors when creating apps through the CLI, starting with the JavaScript / TypeScript version of the Remix app template.
+- Fix create app with old templates. Dev command will work just after the app is created.
```

Comment:
What about re-ordering the list to show new features first?

```suggestion
- ***GraphiQL support***: add a `g` hotkey to dev to open GraphiQL in your browser, using the same credentials as your app
- ***Template flavor support***: add support for different flavors when creating apps through the CLI, starting with the JavaScript / TypeScript version of the Remix app template
- Add support to test locales with functions in Dev mode
- Javascript functions/extensions now support environment variable sustitution
- Allow enabling and disabling dev store preview mode with dashboard managed extensions
- Uuid is no longer generated during app deploy because it's generated in the backend, instead
- Fix: preserve proxy url subpath when running dev
- Fix: javy is only installed when there are Javascript functions
- FIx: running config commands works properly when they are run outside the app root folder
- Fix create app with old templates
```